python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -685,15 +707,11 @@ def __init__(
             dropna=dropna,
         )
 
-    def __getattribute__(self, key):
-        try:
-            return super().__getattribute__(key)
-        except AttributeError:
-            if key in self.obj:
-                return self.obj[key].groupby(
-                    self.grouping, dropna=self._dropna, sort=self._sort
-                )
-            raise
+    def __getattr__(self, key):
+        if key in self.obj:
+            return self.obj[key].groupby(
+                self.grouping, dropna=self._dropna, sort=self._sort
+            )
```

Comment:
Shouldn't we raise an `AttributeError` otherwise?

Reply:
I beat you to it :) See my last commit.